### DISCUSSION TITLE
Is the website and Amaze Cloud Plugin open sorce?

### DISCUSSION THREAD

**Discussion by inson1** [TeamAmaze/AmazeFileManager#4072] [Q&A] [answered]
.

**Comment by VishalNehra** [comment, marked as answer]
No.

**Reply by VishalNehra** [reply]
We're working on making the cloud plugin open source along with OMH. But let's see. Website isn't really a big part so it'll remain closed source.

**Reply by inson1** [reply]
@VishalNehra What is OM?
btw if you are open source project then there should be open source website imho
thanks for the answer
